docs/reference/release-notes/highlights-7.0.0.asciidoc

```diff
@@ -13,5 +13,97 @@ See also <<breaking-changes-7.0,Breaking changes>> and
 //Installation and Upgrade Guide
 
 // tag::notable-highlights[]
+=== Adaptive replica selection enabled by default
 
-// end::notable-highlights[]
\ No newline at end of file
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Skip shard refreshes if a shard is "search idle"
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Default to one shard
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Lucene 8
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Introduce the ability to minimize round-trips in {ccs}
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== New cluster coordination implementation
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Real memory circuit breaker
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== {ccr} is production-ready
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== {ilm} is production-ready
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== {sql} is production-ready
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== High-level REST client is feature-complete
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Support nanosecond timestamps
+
+Words.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Disabling hit counts by default
```

Comment:
this is the breaking part of the change, I think a better description would rather be something like "Faster retrieval of top hits"

Reply:
I pushed b5d322c7912bf14f50cc2a9fc1b25653a9ed5280